pvlib/irradiance.py

```diff
@@ -542,7 +542,8 @@ def poa_components(aoi, dni, poa_sky_diffuse, poa_ground_diffuse):
 def get_ground_diffuse(surface_tilt, ghi, albedo=.25, surface_type=None):
     '''
     Estimate diffuse irradiance from ground reflections given
-    irradiance, albedo, and surface tilt.
+    irradiance, albedo, and surface tilt. The ground is assumed to
+    be isotropic and reflections are Lambertian.
```

Comment:
```suggestion
    be horizontal and flat, and reflections are Lambertian.
```
I don't think the word isotropic is appropriate here.

Reply:
why not?

Reply:
If you want to use the word isotropic I would suggest something more like: the surface is Lambertian and the reflected radiance is isotropic.

Reply:
I can live with that.